routers/users.py

```
@@ -0,0 +1,189 @@
+    if users:
+        new_id = max(u["id"] for u in users) + 1
+
+    # 이미지 파일 저장
+    image_url = None
+    if profileImage:
+        # 파일명 안전하게 생성
+        ext = profileImage.filename.split(".")[-1]
+        safe_filename = f"{new_id}_{int(datetime.now().timestamp())}.{ext}"
+        file_path = os.path.join(UPLOAD_DIR, safe_filename)
+
+        with open(file_path, "wb") as buffer:
```

Ruff (0.14.13)

Async functions should not open files with blocking methods like `open`

(ASYNC230)